README.md

```diff
@@ -1,13 +1,103 @@
-# Automated Surveys with Twilio
+<a href="https://www.twilio.com">
+  <img src="https://static0.twilio.com/marketing/bundles/marketing/img/logos/wordmark-red.svg" alt="Twilio" width="250" />
+</a>
 
-In this example application, you will learn how to create an automated survey that can be completed via SMS text message or voice calls.
+# Automated Surveys. Powered by Twilio - Node.js/Express
 
-[Learn more about this code in our interactive code walkthrough](https://www.twilio.com/docs/howto/walkthrough/automated-survey/node/express).
+Instantly collect structured data from your users with a survey conducted over a voice call or SMS text messages. Learn how to create your own survey in the language and framework of your choice. For a step-by-step tutorial see [twilio docs](https://www.twilio.com/docs/howto/walkthrough/automated-survey/node/express).
 
 ## Deploy On Heroku
 
 [![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy?template=https://github.com/TwilioDevEd/survey-node)
```

Comment:
Seems like this is the perfect time to update the ugly `png` to the gorgeous `svn` button. 😉  What do you think guys?